homeassistant/components/bluetooth/manager.py

```diff
@@ -290,7 +290,7 @@ def _async_check_unavailable(self, now: datetime) -> None:
                     # since it may have gone to sleep and since we do not need an active
                     # connection to it we can only determine its availability
                     # by the lack of advertisements
-                    if advertising_interval := intervals.get(address):
```

Comment:
This check is already running on top of the scanners built in availability list. We shouldn't get here unless the scanner has already removed the device from its availability